### PR TITLE
[simple_kit]: Add colors, texts and themes

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: plugins/simple_kit/lib/src/colors/view/simple_colors.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+abstract class SimpleColors {
+  late Color black;
+  late Color blue;
+  late Color red;
+  late Color green;
+  late Color grey1;
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/colors/view/simple_colors.dart:8]: should we add more informative name? Grey light, grey dark, etc?

File: plugins/simple_kit/lib/src/current_theme_stpod.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+enum STheme { light, dark }
+
+/// State provider for changing theme, defaults to lightTheme
+/// All providers that depend on the current theme must listen for it
+final currentThemeStpod = StateProvider<STheme>(
+  (ref) {
+    return STheme.light;
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/current_theme_stpod.dart:9]: defaults to system's theme?

File: plugins/simple_kit/lib/src/texts/simple_body_text_1.dart [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+import 'base/simple_base_text.dart';
+
+class SBodyText1 extends StatelessWidget {
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/texts/simple_body_text_1.dart:6]: More informative name? (same as previous comment)

File: plugins/simple_kit/lib/src/theme/view/simple_dark_theme.dart [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+import '../../colors/view/simple_colors_dark.dart';
+
+ThemeData sDarkTheme = ThemeData(
+  scaffoldBackgroundColor: SColorsDark().grey5,
+  textTheme: TextTheme(
+    // default text style
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/theme/view/simple_dark_theme.dart:8]: actually, I think this comment is redundant

File: plugins/simple_kit/lib/src/current_theme_stpod.dart (hunk 2 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/current_theme_stpod.dart:9]: At first releases Dark Theme won't be supported from the UI side, I just prepared everything from our side, so we can easily apply it when it takes place. So, right now default theme is `STheme.light`.

File: plugins/simple_kit/lib/src/texts/simple_body_text_1.dart (hunk 3 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/texts/simple_body_text_1.dart:6]: The same as a comment about grey color + this pretty common standard for naming a text like this.

File: plugins/simple_kit/lib/src/theme/view/simple_dark_theme.dart (hunk 4 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/theme/view/simple_dark_theme.dart:8]: There are headline1, headline2, headline3, headline4, headline5, headline6, subtitle1, subtitle2, bodyText1, bodyText2, caption, button, overline parameters that are offered by `TextTheme`, and there is only one that is responsible for default text layout (bodyText2). If you still think that this comment is redundant I can remove it.

**Pull request merged**